app/twitch/add-games-to-list.js

```diff
@@ -0,0 +1,11 @@
+import createGameElement from './create-game-element';
+
+export default function showGamesInList(parentEl, data) {
+  parentEl.innerHTML = '';
+
+  for (let i = 0; i < data.top.length; i++) {
```

Comment:
Once `index.js` is passing in `data.top` then this can just loop through data which will better match the tests.